pmm/aws-staging-start.groovy

```diff
@@ -268,44 +274,54 @@ pipeline {
                 }
                 withCredentials([sshUserPrivateKey(credentialsId: 'aws-jenkins', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
                     sh """
-                        until ssh -i "${KEY_PATH}" -o ConnectTimeout=1 -o StrictHostKeyChecking=no ${USER}@\$(cat IP) date; do
+                        until ssh -i "${KEY_PATH}" -o ConnectTimeout=1 -o StrictHostKeyChecking=no ${USER}@\$(cat IP) 'java -version; sudo yum install -y java-1.8.0-openjdk; sudo /usr/sbin/alternatives --set java /usr/lib/jvm/jre-1.8.0-openjdk.x86_64/bin/java; java -version;' ; do
```

Comment:
Why do we need to install java  in these two places ?

https://github.com/Percona-Lab/jenkins-pipelines/pull/684/files#diff-cf39489e2a5e3b641bec0e09fa1ee7a4f9f4c076eb1b8a7aa517440fb2e22a9cR209
https://github.com/Percona-Lab/jenkins-pipelines/pull/684/files#diff-cf39489e2a5e3b641bec0e09fa1ee7a4f9f4c076eb1b8a7aa517440fb2e22a9cR277

And why do we need to do it at all?  We are connecting to this remote instance  via ssh and it is not jenkins worker. 

Also I do not see a good commit message in this PR. What was the root of issue? What do we try to fix? 

As for me for now it seems as a workaround but not a general fix but I can be wrong. And as I know it is pretty urgent so please update the description and commit and then re-check everything without any rush.